docs/UserGuide.adoc

```diff
@@ -143,20 +143,23 @@ Returns any person having names `Betsy`, `Tim`, or `John`
 Format: `help`
 
 === Importing a list of orders : `import`
-
-You will be able to import orders via .csv files that are given to you by your company.
+==== Description
+You will be able to import orders via csv file that are given to you by your company.
 The CSV file will include all relevant and important details of the parcels; such as the customers’ names,
 transaction id of the parcels and the address of the customers, etc. +
 
-Format: `import FILE NAME`
+==== Format
+* `import FILE_NAME`
 
 ****
-* Import the specific csv with the file name in data folder to the application
-* The FILE NAME should include the extension. For example: *orders.csv*.
-* The CSV file should store at data folder that is at the same directory as the JAR file.
+* Only csv file could be import.
```

Comment:
Should be 'imported'

Reply:
Are you referring to line 155 or any other line?

Reply:
Resolved